GradeCenter.Data/Models/Parent.cs

```diff
@@ -0,0 +1,23 @@
+﻿using GradeCenter.Data.Models.Common;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GradeCenter.Data.Models
+{
+    public class Parent : BaseModel<int>
```

Comment:
I have several points here, nothing scary 

- Inherit from "User" instead
- Please add "First name" and "Last Name" in the "User" class which we will use as base.
- You do not need relation to the School here, instead have a collection of "Students" that will represent his children. :)